ucm2/sof-soundwire/sof-soundwire-init.conf

```diff
@@ -0,0 +1,192 @@
+# the kcontrols initial values, which will be set by `alsactl init`
+
+If.pga1p {
+	Condition {
+		Type ControlExists
+		Control "name='PGA1.0 1 Master Playback Volume'"
+	}
+	True {
+		SectionOnce [
```

Comment:
The name was changed in alsa-lib to SectionBoot:
https://git.alsa-project.org/?p=alsa-lib.git;a=commit;h=14636be4a87ea1cbb13bf8942d07d2066d626e71

.. otherwise looks good.

Reply:
The current name is BootSequence [] .

It may also make sense to put the boot values to the master file (ucm2/sof-soundwire/sof-soundwire.conf) unless the contents is shared with some other configuration files. The contents looks ok.

Reply:
> The name was changed in alsa-lib to SectionBoot:
> https://git.alsa-project.org/?p=alsa-lib.git;a=commit;h=14636be4a87ea1cbb13bf8942d07d2066d626e71
> 
> .. otherwise looks good.

@kv2019i Thanks for reminder. I will update the UCM.

> It may also make sense to put the boot values to the master file (ucm2/sof-soundwire/sof-soundwire.conf) unless the contents is shared with some other configuration files. The contents looks ok.

@perexg OK. I will move it into the master file.